components/ErrorEventsDashboard.js

```diff
@@ -76,19 +77,27 @@ function ErrorEventsDashboard () {
         : 'No errors reported in the last two weeks.'
       }
       <style jsx>{`
```

Comment:
Might be nice to have pagination controls at the bottom too in case the user scrolls to the bottom of the list and then wants to go to the next page. Currently, they'd then have to scroll up to the top to click the button to get to the next page.